Source/WebCore/rendering/svg/SVGResources.cpp

```diff
@@ -236,7 +236,8 @@ std::unique_ptr<SVGResources> SVGResources::buildCachedResources(const RenderEle
             else if (!renderer.document().settings().layerBasedSVGEngineEnabled())
                 treeScope.addPendingSVGResource(id, element);
 #else
-            treeScope.addPendingSVGResource(id, element);
+            else
```

Comment:
This is no longer needed, LBSE doesn't use SVGResources anymore.

Reply:
Good point, removed.